test/unit/alphabet/nucleotide/dna3bs_test.cpp

```diff
@@ -0,0 +1,98 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+#include <seqan3/alphabet/concept.hpp>
+#include <seqan3/alphabet/nucleotide/all.hpp>
```

Comment:
why do you include all and not only dna3bs?

Reply:
oops, leftover from the nucleotide integration test I copied this from :innocent: